doc/openocd.texi

```diff
@@ -11369,16 +11369,26 @@ OpenOCD. When off, they generate a breakpoint exception handled internally.
 The commands below can be used to prevent OpenOCD from using certain RISC-V trigger features.
 For example in cases when there are known issues in the target hardware.
 
-@deffn {Command} {riscv set_enable_eq_match_trigger} [on|off]
-When on (default), allow OpenOCD to use exact-match triggers in watchpoints.
-@end deffn
+@deffn {Command} {riscv set_enable_trigger_feature} [(@option{eq}|@option{napot}|@option{ge_lt}|@option{all}) (@option{wp}|@option{none})]
```

Comment:
A description of what happens when `riscv set_enable_trigger_feature` is called without arguments should be added.

Reply:
Addressed.